package ch2test;

import java.io.IOException;

public class test_p24 {
	public static void main(String[] args)throws IOException
	{
		for (int i=1;i<=5;i++)
		{
			System.out.println("第"+i+"次的迴圈");
		}
		System.out.println("迴圈結束");
	}
}
